README.md

```diff
@@ -100,28 +104,36 @@ Name     | Official | Tested  | Compatible | Patched
 Electron | 22       |         |            |
 Go       | 1.20     |         | 1.21.4     |
 Node.js  | 12.22.12 | 16.18.1 | 19.9.0     |
-Python   | 3.8.18   |         |            | 🇵 3.12.2
+Python   | 3.8.19   |         |            | 🇵 3.12.3
+Java     | 13       |         | 22         |
 
-- [PythonWin7](https://github.com/adang1345/PythonWin7)
+- [PythonWin7](https://github.com/adang1345/PythonWin7/releases)
```

Comment:
There are no releases here, the page is empty, the installers are in the repo tree.